### PR TITLE
use deb package for debian install

### PR DESCRIPTION
**Pull request by blacksponge**
## Purpose

Provide package installation

## Context

https://github.com/zonemaster/zonemaster-cli/pull/233

## Changes

Debian installation use package repository

## How to test this PR

Follow the installation instruction for debian.

### CODE REVIEW

**Review by matsduf — APPROVED** [review]

**Review by matsduf — APPROVED** [review]
I approve.

File: docs/Installation.md [hunk 1]
```diff
@@ -73,7 +74,29 @@ Zonemaster::Engine, see the [declaration of prerequisites].
...
+Using pre-built packages is the preferred method for Debian. If you prefer to
+install from CPAN instead, follow the steps for Ubuntu.
+
+1) Upgrade to latest patch level
+
+   ```sh
+   sudo apt update && sudo apt upgrade
+   ```
+
+2) Add Zonemaster packages repository to repository list
```
Comment (matsduf) [inline docs/Installation.md:88]: I guess "unless already done", and that applies to CLI too, but that could be adjusted for the release in 2022.

Comment (blacksponge) [reply docs/Installation.md:88]: Yes I guess that could be adjusted later as we might need to update the instruction for the upgrade process. As for now, executing the script multiple times doesn't do any harm.

Comment (matsduf) [reply docs/Installation.md:88]: I agree. Let us get some experience. I just wanted to comment.

**Review by pnax — APPROVED** [review]

**Review by mattias-p — APPROVED** [review]

**Pull request merged**